emperor/support_files/js/util.js

```diff
@@ -83,6 +82,34 @@ function naturalSort(list){
   return result.concat(alphaPart, numericPart);
 }
 
+
+/**
+ *
+ * Utility function that splits the lineage into taxonomic levels
+ * and returns the taxonomic level specified
+ *
+ * @param {node} XML DOM object, usually as created by the document object.
+ *
+ * @return string representation of the node object.
+ *
+*/
+function truncateLevel(lineage, level_index){
+    if (level_index==0){
+	return lineage;
+    }
+    var levels = lineage.split(';');
+    var taxa_label = '';
+    for( var i = 0; (i<level_index && i<levels.length); i++){
+        var level = levels[i];
```

Comment:
The indentation of this for block is a bit funny, would you mind fixing that and making sure it is all two-spaced?